d3d12/.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
```

Comment:
This shouldn't be included

Reply:
I think cribbing CI from `wgpu` would be an overall improvement, so I see not reason not to do this for now. 🙂

Done with 6d355881.